components/script/dom/filereader.rs

```diff
@@ -245,7 +272,28 @@ impl FileReader {
             format!("data:{};base64,{}", data.blobtype, base64)
         };
 
-        DOMString::from(output)
+        StringOrObject::String(DOMString::from(output))
+    }
+
+    // https://w3c.github.io/FileAPI/#dfn-readAsArrayBuffer
+    #[allow(unsafe_code, unused_variables)]
+    fn perform_readasarraybuffer(cx: *mut JSContext, data: ReadMetaData, bytes: &[u8])
```

Comment:
There might be more than a few logical errors in this function, as I haven't asked anyone to look at this piece of code since the compiler didn't complain.